docs/Breaking API Changes.md

```diff
@@ -64,3 +64,10 @@ Roslyn does not support implementing completion for arbitrary languages.
 ### `Microsoft.CodeAnalysis.CodeStyle.NotificationOption` is now immutable
 
 All property setters now throw an exception.
+
+# Version 4.4.0
+
+`Workspace.OnWorkspaceFailed` is no longer called when an error occurs while reading source file content from disk.
```

Comment:
Was this necessary fallout, or a "while I'm here, get rid of this bad thing too"?

Reply:
We didn't mark the event as obsolete?

Reply:
No, the event is still valid and used e.g. in MSBuildWorkspace. It's just not used for the specific failure when source file content can't be loaded.